packages/experimental/Avatar/src/__tests__/JSAvatar.test.jsx

```diff
@@ -29,130 +28,108 @@ describe('Avatar rendering', () => {
   });
 });
 
-describe('validateAlphabeticalCharacters method', () => {
-  it('validates alphabetical characters', () => {
-    for (let i = 0; i < alphabeticalTestNames.length; i++) {
-      expect(validateAlphabeticalCharacters(alphabeticalTestNames[i])).toBeTruthy();
-    }
-    for (let i = 0; i < otherTestNames.length; i++) {
-      expect(validateAlphabeticalCharacters(otherTestNames[i])).toBeFalsy();
-    }
+describe('getInitials - names with titles', () => {
+  it('returns initials without title', () => {
+    expect(getInitials('Mr Faynman')).toBe('F');
   });
+  it('returns initials for names with multi-word titles', () => {
+    expect(getInitials('Consul General Richard Faynman')).toBe('RF');
+    expect(getInitials('Major General Victor')).toBe('V');
+  });
+});
 
-  it("doesn't validate undefined name", () => {
-    expect(validateAlphabeticalCharacters(undefined)).toBeFalsy();
+describe('getInitials method - language support', () => {
+  it('returns correct initials cyrillic language', () => {
+    const test = getInitials('Илон Маск');
+    expect(test).toBe('ИМ');
   });
-  it("doesn't validate an empty string", () => {
-    expect(validateAlphabeticalCharacters('')).toBeFalsy();
+  it('calculates correct initials for non-ASCII characters', () => {
+    const test = getInitials('Írissa Þórðardóttir');
+    expect(test).toBe('ÍÞ');
   });
-  it("doesn't validate null name", () => {
-    expect(validateAlphabeticalCharacters(null)).toBeFalsy();
+  it('returns correct initials for non-ASCII characters', () => {
+    const test = getInitials('Øyvind Åsen');
+    expect(test).toBe('ØÅ');
   });
 });
 
-describe('removeTitlesFromName method', () => {
-  it('removes titles from the name', () => {
-    const words = ['Mr', 'Faynman'];
-    expect(removeTitlesFromName(words)).toStrictEqual(['Faynman']);
-  });
-  // it('removes multiword titles from the name', () => {
-  //   const name = 'Consul General Richard Faynman';
-  //   expect(removeTitlesFromName(name)).toStrictEqual(['Richard Faynman']);
-  // });
+describe('getInitials method - RTL', () => {
+  it('returns correct initials in RTL', () => {
+    const test = getInitials('Richard Feynman', true);
+    expect(test).toBe('FR');
+  });
```

Comment:
Wait... I don't think you want to flip an LTR language in an RTL context. So even if the app is in an RTL context (Let's say, Arabic), an Avatar representing the English name "Richard Feynman" should be "RF". This test feels misleading?

Reply:
It would be good to have test in Arabic but I had to check it on the language I understand. I'll update this test when I add support for Arabic language

Reply:
If it helps:
My name in Arabic: سعد نيمي
My initials:  س ن

Reply:
I believe that for Arabic and Asian languages they only show one character in the PCX logic. Ther's not really space for more than one character to show.

Reply:
Thank you :)